Earthfile

```diff
@@ -1,4 +1,6 @@
 VERSION 0.6
+ARG go_version=1.19.4
```

Comment:
Be consistent in value assignment - in one place you don't have an assignment operator, in another you do.

```earthly
ARG go_version=1.19.4
// vs
ARG go_version 1.19.4
```